web/client/components/map/openlayers/__tests__/Feature-test.jsx

```diff
@@ -101,7 +101,8 @@ describe('Test Feature', () => {
 
         expect(layer).toExist();
         // count layers
-        expect(container.getSource().getFeatures().length === 1 );
+        const count = container.getSource().getFeatures().length;
```

Comment:
Can you revert this? Nothing changed

Reply:
ok,
I add this
```expect(container.getSource().getFeatures().length ).toBe(1);```
because
```expect(container.getSource().getFeatures().length === 1 );```
it seems not to work

Reply:
even if I put === 0 the test does not fail

Reply:
You are right, the old test was wrong, it misses a toBe(...)